frappe/utils/backups.py

```diff
@@ -102,15 +102,17 @@ def get_recent_backup(self, older_than):
 
 		for this_file in file_list:
 			this_file = cstr(this_file)
+			file_name = os.path.splitext(this_file)[0]
 			this_file_path = os.path.join(get_backup_path(), this_file)
+
 			if not is_file_old(this_file_path, older_than):
-				if "-private-files" in this_file_path:
-					backup_path_private_files = this_file_path
-				elif "-files" in this_file_path:
-					backup_path_files = this_file_path
-				elif "-database" in this_file_path:
+				if file_name.endswith("{}-database.sql".format(self.site_slug)):
 					backup_path_db = this_file_path
-				elif "site_config" in this_file_path:
+				elif file_name.endswith("{}-files".format(self.site_slug)):
```

Comment:
public files will require to have a `-public-files` slug now: ef4a28e10c0de0083c57278b615e587600b64e3b

Reply:
just saw it has been reverted.